src/App.tsx

```diff
@@ -1,61 +1,133 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
```

Comment:
Ensure that data is prepared in the `App` component and passed to child components, avoiding the use of `import` in non-root components .